ctru-rs/src/applets/swkbd.rs

```diff
@@ -336,6 +337,30 @@ impl SoftwareKeyboard {
         self.state.max_digits = digits;
     }
 
+    /// Set the initial text for this software keyboard.
+    ///
+    /// The initial text is the text already written when you open the software keyboard.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # let _runner = test_runner::GdbRunner::default();
+    /// # fn main() {
+    /// #
+    /// use ctru::applets::swkbd::SoftwareKeyboard;
+    /// let mut keyboard = SoftwareKeyboard::default();
+    ///
+    /// keyboard.set_initial_text("Write here what you like!");
+    /// #
+    /// # }
+    #[doc(alias = "swkbdSetInitialText")]
+    pub fn set_initial_text(&mut self, text: &str) {
+        unsafe {
+            let nul_terminated: String = text.chars().chain(once('\0')).collect();
```

Comment:
You should probably use CString for a more idiomatic conversion.

Reply:
Agreed, I think this could just say `CString::new(text).unwrap()`

Reply:
i just copied what set_hint_text does, as the signature is the same